modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -471,6 +469,30 @@ inline Ort::Value createTensor(const Ort::MemoryInfo& memory_info,
         return createTensor<float>(memory_info, tensor_params, data);
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32:
         return createTensor<int32_t>(memory_info, tensor_params, data);
+    case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64:{
```

Comment:
Can we get rid of `temp` vector here? I assume `convertInt32ToInt64` can write straight to `i64_tensor` memory. You will avoid extra copy by doing this.

Reply:
Ok, fixed